app/models/pacbio/smrt_link_version.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Pacbio
+  # Pacbio::SmrtLinkVersion
+  class SmrtLinkVersion < ApplicationRecord
+    validates :name, presence: true, uniqueness: true, format: Version::FORMAT
+
+    has_many :smrt_link_option_versions, class_name: 'Pacbio::SmrtLinkOptionVersion',
+                                         foreign_key: :pacbio_smrt_link_version_id,
+                                         dependent: :destroy, inverse_of: :smrt_link_version
+    has_many :smrt_link_options, through: :smrt_link_option_versions, source: :smrt_link_option,
+                                 class_name: 'Pacbio::SmrtLinkOption'
+
+    # This is the other side of the belongs_to association in pacbio runs.
+    has_many :runs, class_name: 'Pacbio::Run', dependent: :destroy, inverse_of: :smrt_link_version
```

Comment:
We don't. We put that in because we were getting all sorts of failures with regard to reverse relationships and rubocop. I would prefer it not to be there at all but we couldn't find an easy way to fix it.

Reply:
If its just there for Rubocop, then you could add the Rubocop comment to ignore the next line